src/jshint.js

```diff
@@ -3236,7 +3236,13 @@ var JSHINT = (function() {
             error("E034");
           }
 
-          i = propertyName();
+          // Checks if the getter and/or setter is a computed name by
+          // checking that the next value of the getter and/or setter is a [.
```

Comment:
This comment is a bit literal; on its own, the code already communicates this information.